config/routes.rb

```diff
@@ -17,6 +17,7 @@
   namespace :api do
     namespace :v1 do
       get 'events', to: 'event#index'
+      get 'campaigns', to: 'campaign#index'
```

Comment:
oh... just noticed one thing. You are using singular controllers :-o :-o :-o ... typically in rails controllers for resources are plural. campaings_controller, etc.